### DISCUSSION TITLE
how to add TheHive after initially declining in setup

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2784] [Unsupported Versions]
I initially declined to install TheHive at setup because of repeated failures with no functioning solutions found to push it to completion at that stage.  I noticed when doing a docker-refresh that TheHive containers are all in the system even though I chose not to install it.  Is it possible to add TheHive functionality to an SO Manager that's been running for a while after it was initially declined?

**Comment by dootdootbeepbeep** [comment]
I'm wondering the same thing..
